src/metabase/driver/mysql.clj

```diff
@@ -38,7 +40,11 @@
 
 (defmethod driver/display-name :mysql [_] "MySQL")
 
-(defmethod driver/database-supports? [:mysql :nested-field-columns] [_ _ _] true)
+(defmethod driver/database-supports? [:mysql :nested-field-columns] [_ _ database]
+  (let [json-setting (get-in database [:details :json-unfolding])
```

Comment:
couldn't we just do `(or (get-in database [:details :json-unfolding]) true)`?

Reply:
we could, it would also be clearer. will do